### DISCUSSION TITLE
Windows notifications covering system tray

### DISCUSSION THREAD

**Discussion by ishalev** [valinet/ExplorerPatcher#1908] [General]
See image below or this link https://screenrec.com/share/UvklZbMLot -- is this related to EP? Any way to resolve this so that the notification doesn't overlap the system tray?

Edited to add -- the notifications never fade, even though they are set to fade after 5 seconds

![image](https://github.com/valinet/ExplorerPatcher/assets/6796314/ec9da594-1d48-430d-9315-bb7c87cd8de4)

**Comment by pyrates999** [comment]
duplicate of: https://github.com/valinet/ExplorerPatcher/issues/1759

Please close this one.
